src/physics_body.js

```diff
@@ -0,0 +1,35 @@
+import Vec2 from 'vec2';
```

Comment:
The convention i've been using for files has been separating by dashes.  `physics-body.js`  This follows most of what I see on npm.... I don't know why.